src/ol/ol.js

```diff
@@ -160,6 +174,40 @@ ol.OVERVIEWMAP_MIN_RATIO = 0.1;
 
 
 /**
+ * @define {number} Maximum number of source tiles for raster reprojection of
+ *     a single tile.
+ *     If too many source tiles are determined to be loaded to create a single
+ *     reprojected tile the browser can become unresponsive or even crash.
+ *     This can happen if the developer defines projections improperly and/or
+ *     with unlimited extents.
+ *     If too many tiles are required, no tiles are loaded and
+ *     `ol.TileState.ERROR` state is set. Default is `100`.
+ */
+ol.RASTER_REPROJ_MAX_SOURCE_TILES = 100;
```

Comment:
Same as above, and the same for the defines below: `RASTER_REPROJECTION_` instead of `RASTER_REPROJ_`.